Engine/src/Renderer/RHI/RendererAPI.cpp

```diff
@@ -1,9 +1,15 @@
 #include "RendererAPI.h"
 
 #include "Platform/OpenGL/OpenGLRendererAPI.h"
+#include "Platform/OpenGL41/OpenGL41RendererAPI.h"
 
 namespace elv {
+// TODO: change to either runtime check or use more generic factor for checking
+#ifdef __APPLE__
+RendererAPI::API RendererAPI::s_API = RendererAPI::API::OpenGL41;
+#else
 RendererAPI::API RendererAPI::s_API = RendererAPI::API::OpenGL;
+#endif
```

Comment:
See todo.

Reply:
This is enough for now. We can add more specific logic to choose API in the future.